courses/free-courses-en.md

```diff
@@ -148,6 +148,7 @@
 ### AutoIt
 
 * [AutoIt Scripting Tutorial](https://www.youtube.com/playlist?list=PL4Jcq5zn02jKpjX0nqI1_fS7mEEb5tw6z) - TutsTeach (YouTube)
+* [AutoIt Turtorial](https://www.youtube.com/watch?v=82ifMPijghw&list=PL1DCD109B801D0DE6) - 403forbidden403 (Youtube)
```

Comment:
We should fix this acording to CONTRIBUTE guidelines
- Use playlist url
- Fix wrong title

Maybe this hunk solve all:
```markdown
* [AutoIt Tutorials](https://www.youtube.com/playlist?list=PL1DCD109B801D0DE6) - 403forbidden403 (YouTube)
```

Reply:
thanks